src/ui/ResultList/ResultList.ts

```diff
@@ -624,9 +624,9 @@ export class ResultList extends Component {
       return;
     }
 
-    if (this.options.infiniteScrollContainer instanceof Window) {
+    if (typeof this.options.infiniteScrollContainer.scrollTo === 'function') {
       const win = <Window>this.options.infiniteScrollContainer;
-      const searchInterfacePosition = win.pageYOffset + this.searchInterface.element.getBoundingClientRect().top;
+      const searchInterfacePosition = window.pageYOffset + this.searchInterface.element.getBoundingClientRect().top;
```

Comment:
What should we use `window` instead `win` (aka the scrollContainer) for the offset? Is this a second bug?

Reply:
pageYOffset is only present on Window.

Reply:
Not 100% sure I understand what you mean by that.

I agree the code/typecast was kind of meaningless before, and you change seems fine, but still I'm not sure I understand your comment.

In the previous code it was accessing pageYOffset on the `win` variable (that is cast as a Window).

`window (the object)` != `Window (the interface)` ?

Reply:
@jeremierobert-coveo, so if it was only on `window`, then using `win` returns `undefined`? So this means it was a bug before?

In that case, would it be possible to use the `scrollTo` method on window? If so, wouldn't that be more consistent instead of jumping between `window` and `infiniteScrollContainer`?